app/context/userContext.tsx

```diff
@@ -10,6 +10,7 @@ import { JsonRpcSigner, Web3Provider } from "@ethersproject/providers";
 // -- Ceramic and Glazed
 import { EthereumAuthProvider } from "@self.id/web";
 import { useViewerConnection } from "@self.id/framework";
+import { DIDSession } from "@glazed/did-session";
```

Comment:
Noticed you're using `@glazed/did-session` here--was there a specific reason you used it and not `did-session`?

Reply:
That is the library as it is registered with npm, and this is what we use: https://www.npmjs.com/package/@glazed/did-session.
Not sure why the non-prefixed one also exists, as it seems to be the same version: https://www.npmjs.com/package/did-session